### PR TITLE
Chore/update libs

### PR DESCRIPTION
**Pull request by jduffy3**

### CODE REVIEW

File: src/components/WarningHighlight.js [hunk 1]
```diff
@@ -1,15 +1,20 @@
...
+      data-tooltip-content={props.message}
+      data-tooltip-place={props.position}
+      data-tooltip-position-strategy="fixed"
     >
-      <WarningTooltip />
+      <Tooltip
+        className="jns-tooltip"
+        anchorSelect=".jns-highlight"
+        float={true}
+      />
```
Comment (sbrudz) [inline src/components/WarningHighlight.js:17]: In GMail, the message for "does this make sense" is overflowing the box:

image

Maybe get rid of `max-height` from jns-tooltip CSS?

File: src/components/WarningTooltip.js [hunk 2]
```diff
@@ -1,14 +0,0 @@
-import { h } from 'preact';
-import ReactTooltip from 'react-tooltip';
-
-export default function WarningTooltip() {
-  return (
-    <ReactTooltip
-      class="jns-tooltip"
-      type="dark"
-      effect="float"
-      border={true}
```
Comment (sbrudz) [inline src/components/WarningTooltip.js:10]: We added a thin white border to the tooltip box to improve visibility when GMail is in dark mode.  It looks like React Tooltip v5 says to use CSS for this effect.

File: src/components/WarningHighlight.js (hunk 1 above)
Comment (sbrudz) [reply src/components/WarningHighlight.js:17]: Actually, the white text is there, but the background color doesn't extend all of the way down:
image

**Review by sbrudz — CHANGES_REQUESTED** [review]
Looks good overall.  Some small visual tweaks to the tooltip are needed.  The `data-testid` stuff is optional.

File: src/components/WarningTooltip.js (hunk 2 above)
Comment (jduffy3) [reply src/components/WarningTooltip.js:10]: Thanks for catching this!

File: just-not-sorry.css [hunk 3]
```diff
@@ -4,7 +4,5 @@
 
 .jns-tooltip {
   max-width: 380px;
-  max-height: 100px;
-  margin-bottom: 30px;
-  margin-top: 50px;
+  border: solid white;
```
Comment (sbrudz) [inline just-not-sorry.css:7]: The border is a little bold.  How about this instead?
```suggestion
  border: 1px solid darkgray;
```

**Review by sbrudz — APPROVED** [review]
Looks good - Thank you!

**Pull request merged**